src/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -244,7 +245,7 @@
   private   QuickAttachmentDrawer  quickAttachmentDrawer;
   private   InputPanel             inputPanel;
   private   GestureDetector        swipeDetector;
-  private   View.OnTouchListener   swipeListener;
```

Comment:
Did we change the scope for a specific reason?

Reply:
Yeah, it IntelliJ realized it could be in local scope.